android/src/main/java/ch/milosz/reactnative/RNZoomUsInMeetingServiceListener.java

```diff
@@ -0,0 +1,297 @@
+import com.facebook.react.modules.core.DeviceEventManagerModule;
+
+import java.util.List;
+
+// import us.zoom.sdk.FreeMeetingNeedUpgradeType;
+import us.zoom.sdk.InMeetingAudioController;
+import us.zoom.sdk.InMeetingChatMessage;
+import us.zoom.sdk.InMeetingEventHandler;
+import us.zoom.sdk.InMeetingService;
+import us.zoom.sdk.InMeetingServiceListener;
+import us.zoom.sdk.InMeetingUserInfo;
+
+public class RNZoomUsInMeetingServiceListener implements InMeetingServiceListener {
+    private final static String TAG = "RNZoomUsInMeetingServiceListener";
+    private final ReactApplicationContext reactContext;
+    private final InMeetingService inMeetingService;
+
+    public RNZoomUsInMeetingServiceListener(ReactApplicationContext context, InMeetingService inMeetingService) {
+        reactContext = context;
+        this.inMeetingService = inMeetingService;
+    }
+
+    private void notifyEvent(String event, WritableMap params) {
+        WritableMap map = Arguments.createMap();
+        map.putString("event", event);
+        map.putMap("payload", params);
+        reactContext.getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class)
+                .emit("InMeetingEvent", map);
+    }
+
+    private WritableMap extractUserInfoFromUserId(long userId) {
+        InMeetingUserInfo userInfo = inMeetingService.getUserInfoById(userId);
+        WritableMap map = Arguments.createMap();
+        map.putString("name", userInfo.getUserName());
+        map.putString("userId", "" + userId);
+        // map.putString("participantId", userInfo.getParticipantID());
+        return map;
+    }
+
+    @Override
+    public void onWebinarNeedRegister() {
+
+    }
+
+    @Override
+    public void onMeetingNeedPasswordOrDisplayName(boolean b, boolean b1, InMeetingEventHandler inMeetingEventHandler) {
+
+    }
+
+    @Override
+    public void onJoinWebinarNeedUserNameAndEmail(InMeetingEventHandler inMeetingEventHandler) {
+
+    }
+
+    @Override
+    public void onMeetingNeedColseOtherMeeting(InMeetingEventHandler inMeetingEventHandler) {
+
+    }
+
+    @Override
+    public void onMeetingFail(int i, int i1) {
+
+    }
+
+    @Override
+    public void onMeetingLeaveComplete(long l) {
+    }
+
+    @Override
+    public void onMeetingUserJoin(List<Long> list) {
+        WritableMap map = Arguments.createMap();
+        WritableArray userList = Arguments.createArray();
+        for (long l: list) {
+            userList.pushMap(extractUserInfoFromUserId(l));
+        }
+        map.putArray("userList", userList);
+        notifyEvent("meeting.user.joined", map);
+    }
+
+    @Override
+    public void onMeetingUserLeave(List<Long> list) {
+        WritableMap map = Arguments.createMap();
+        WritableArray userList = Arguments.createArray();
+        for (long l: list) {
+            userList.pushMap(extractUserInfoFromUserId(l));
+        }
+        map.putArray("userList", userList);
+        notifyEvent("meeting.user.left", map);
```

Comment:
Is this of the same structure as on ios side?
It looks to me that there you are sending an object, and here you are sending array under `userList` key